keyboards/synthlabs/065/info.json

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
Please run `qmk format-json` on this file.

Reply:
Ok, I ran it and it just outputted to terminal? Copied the output and pasted it into the file though. Hopefully I did that all correctly - the whitespace looks more consistent now.